### ISSUE TITLE
[BUG] list of tuple of single item is parsed but take only the first character of each token get parsed

### ISSUE THREAD

**Issue by OrHayat** [BrianPugh/cyclopts#105]
```python
import cyclopts
from cyclopts import Parameter
from typing import Annotated

app = cyclopts.App(converter=cyclopts.convert)


@app.command(name="create")
def create_dict(  
    numbers: Annotated[list[tuple[str]],Parameter(name="--item-int")],
    strs: Annotated[list[tuple[str]],Parameter(name="--item")]):
    """
    item:
        key,value of items to store in the dict - can pass more then one pair or pass the flag multiple times
    """
    # res=dict(items)
    print(f"numbers={numbers}, strs={strs}",)


if __name__=="__main__":
    app()
```

```bash
> poetry run python3 ./test.py   create --item aa --item-int 16
numbers=[('1',)], strs=[('a',)]
```

**Comment by BrianPugh** [comment]
I opened up PR #106 that should resolve this issue. With that said, I think in  your example:

1. You probably meant `numbers: Annotated[list[tuple[int]],Parameter(name="--item-int")],`
2. Having a tuple with a single string element isn't common, but it should certainly work as expected with PR #106

**Comment by BrianPugh** [comment]
This is fixed in v2.3.1. If you believe it's not fully fixed we can re-open the issue. Thanks!
